pkg/setting/setting.go

```diff
@@ -367,6 +367,7 @@ type Cfg struct {
 	CheckForUpdates      bool
 	ReportingDistributor string
 	ReportingEnabled     bool
+	ApplicationInsightsInstrumentationKey string
```

Comment:
Uber nit: can we place it together with other tracking services keys? Meaning line 146 :)

Reply:
Thanks for feedback! In theory I agree, but I think all the variables above are global, and the idea is that we're suppose to use the `Cfg` struct instead.